build/lib/transaction/send/fee/actual-transaction-fee-box.d.ts

```diff
@@ -1,11 +1,12 @@
+/// <reference path="../../../../../src/react-jss.d.ts" />
 import * as React from 'react';
-import { StyleSheet, StyledComponentProps } from 'react-jss';
+import { StyleSheet } from 'react-jss';
 export declare const styles: StyleSheet;
 export declare type ActualTransactionFeeBoxProps = {
     locale: string;
     ethFee: number;
     usdFee: number;
     fiatCurrency: string;
 };
-export declare const ActualTransactionFeeBox: React.ComponentType<ActualTransactionFeeBoxProps & StyledComponentProps<string>>;
+export declare const ActualTransactionFeeBox: React.ComponentType<ActualTransactionFeeBoxProps & import("react-jss").StyledComponentProps<string>>;
```

Comment:
In general it's a bad practice to add import in the middle of a document, but if it works for you on lint I don't have a strong opinion about that.

Just test it well, the issue that might occur will be on runtime in the wallet with this.